scripts/install/ShowQtiAdaptiveSectionIds.php

```diff
@@ -37,9 +38,9 @@ class ShowQtiAdaptiveSectionIds extends InstallAction
 {
     public function __invoke($params)
     {
-        $adaptiveSectionIdsProperty = new \core_kernel_classes_Property(CatService::CAT_ADAPTIVE_IDS_PROPERTY);
-        $widgetProperty = new \core_kernel_classes_Property(WidgetRdf::PROPERTY_WIDGET);
-        $adaptiveSectionIdsProperty->editPropertyValues($widgetProperty, WidgetDefinitions::PROPERTY_JSONOBJECT);
+        $adaptiveSectionIdsProperty = new KernelProperty(CatService::CAT_ADAPTIVE_IDS_PROPERTY);
```

Comment:
We have `OntologyAwareTrait`, maybe it makes sense to use its methods to create property objects?

Reply:
It makes perfect sense, thanks!